resource/build2019-prod/rules.json

```diff
@@ -0,0 +1,110 @@
+{
+  "UserAccess": [
+    {
+      "Identifiers": ["Anonymous"],
+      "Access": {
+        "SessionCatalog": true,
+        "MyIgnite": false,
+        "ScheduleBuilder": false,
+        "MobileApp": false,
+        "MeetingScheduler": false,
+        "MobileAppMessaging": false,
+        "LocationBasedMessaging": false
+      },
+      "VisibleAttendeeTypes": [],
+      "Reporting": {
+        "SessionCounts": false
+      },
+      "EvalAccess": [],
+      "SessionSetAccess": [
+        {
+          "ApplicableFrom": null,
+          "ApplicableTo": null,
+          "Identifier": "Session Catalog"
+        }
+      ]
+    },
+    {
+      "Identifiers": [
+        "Comp Academic",
+        "Comp General Attendee",
+        "General Attendee Cross Charge",
+        "Paid General Attendee Beyond Capacity",
+        "Paid General Attendee Full Price",
+        "Paid General Attendee Partial",
+        "Student Cross Charge",
+        "Pre-Registered Attendee",
+        "Comp Microsoft Attendee",
+        "Microsoft Attendee Cross Charge",
+        "Paid Microsoft Attendee",
+        "Paid Microsoft Attendee Beyond Capacity",
+        "Staff Chaperone /ADA Support",
+        "Staff Microsoft",
+        "Financial Analyst",
+        "Speaker External",
+        "Speaker Microsoft",
+        "Staff External",
+        "Staff Partner Booths",
+        "Analyst",
+        "Press",
+        "Press Staff",
+        "Crew"
+      ],
+      "Access": {
+        "SessionCatalog": true,
+        "MyIgnite": true,
+        "ScheduleBuilder": true,
+        "MobileApp": true,
+        "MeetingScheduler": true,
```

Comment:
let's make this false please, we're not doing meetings on this show, so setting this correctly is good practice